api/v1alpha1/eraserconfig_types.go

```diff
@@ -76,7 +76,7 @@ func (r *Runtime) UnmarshalJSON(b []byte) error {
 }
 
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+// NOTE: yaml tags are required.  Any new fields you add must have yaml tags for the fields to be serialized.
```

Comment:
i thought we reverted this change

Reply:
We did, so this should be reverted here too.